.github/workflows/docs.yml

```diff
@@ -0,0 +1,26 @@
+name: "Documentation build check"
+on:
+  pull_request:
+    branches:
+      - "*"
+
+jobs:
+  docs:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+        with:
+          submodules: 'recursive'
+      - uses: actions/setup-python@v1
+        with:
+          python-version: "3.8"
```

Comment:
Use py36 for all non-unit jobs as this is the most stable and widlely supported version. py38 is known to keep causing problems with various platforms and libraries.

Reply:
My plan was to propose a single change that enable all currenlty travis jobs using github workdlows but I had not time to do it yet.

Reply:
Switch to the same version of python, like on read the docs.